teleport-bootstrap-script/metadata.tpl

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+# Config /etc/teleport
+
+## Get private IP for advertise_ip
+export ADVERTISE_IP="$(ip addr show eth0 | grep 'inet ' | awk '{print $2}' | cut -f1 -d'/')"
```

Comment:
if you want you could replace this with `curl http://169.254.169.254/2009-04-04/meta-data/local-ipv4`

Reply:
unless this is also for other providers

Reply:
yes I didn't get it from metadata because it's indeed intended to work with other providers too